tests/existing-resources/README.md

```diff
@@ -0,0 +1,3 @@
+# Existing Resources For use in tests
```

Comment:
```suggestion
# Existing resources to use in tests
```

Reply:
Just a capitalization comment. See Conall's comments about what the title should be.